internal/source/code.go

```diff
@@ -54,25 +54,31 @@ func Mutate(raw string, a []Edit) (string, error) {
 	if len(a) == 0 {
```

Comment:
I was having a bit of trouble with this, so I tested the heck out of it and these were the resulting edits.

Reply:
The edit code is super hacky so I'm not surprised you needed to add all these tests. Hopeful in the future we can get rid of all of this code and generate SQL using an AST.